scripts/docs/snippetadapter.mjs

```diff
@@ -5,60 +5,17 @@
 
 /* eslint-env node */
 
-import upath from 'upath';
 import fs from 'fs';
-import module from 'module';
 import { fileURLToPath } from 'url';
-import minimatch from 'minimatch';
-import webpack from 'webpack';
-import { bundler, loaders, tools } from '@ckeditor/ckeditor5-dev-utils';
-import { CKEditorTranslationsPlugin } from '@ckeditor/ckeditor5-dev-translations';
-import MiniCssExtractPlugin from 'mini-css-extract-plugin';
-import TerserPlugin from 'terser-webpack-plugin';
+import { createRequire } from 'module';
```

Comment:
Only this file has some notable changes. Changes in all other files should be:

* typo fixes,
* import fixes,
* removal of license keys from snippets.